### ISSUE TITLE
Selinux_Changer

### ISSUE THREAD

**Issue by alicee98** [Magisk-Modules-Alt-Repo/submission#95]
**Name**: Selinux Changer
**Description**:  Magisk Module For Change The Selinux Permission To 2 Mode. Its Enforcing And Permissive
**Link**: https://github.com/alicee98/Selinux_Changer

**Comment by HuskyDG** [comment]
SELinux Permissive is only meant for debug. You don't know how dangerous it was...
I think author of sound modules should address this problem by adding minimal allow rules in `sepolicy.rule` instead of turn SELinux into permissive.
TIP: Denials SELinux can be catched by logcat

**Comment by HuskyDG** [comment]
Your module atleast should add some warning for users because most users do not know about the dangerous of SELinux Permissive. They only know the immediate benefits.

**Comment by alicee98** [comment]
> SELinux Permissive is only meant for debugging and should not use for daily uses. SELinux Permissive open a door for apps to abuse security holes. You don't know how dangerous it was... I think author of sound modules should address this problem by adding minimal allow rules in `sepolicy.rule` instead of turn SELinux into permissive. TIP: Denials SELinux can be catched by logcat

I has Added It In V2 Version. I added It In customise.sh And in changelog.md
Link to V2 tag realease : https://github.com/alicee98/Selinux_Changer/releases/tag/V2

**Comment by HuskyDG** [comment]
hmmm, why do you remove magisk logs here?

https://github.com/alicee98/Selinux_Changer/blob/adfe2f29663cf2474fd80f6e22bc31e4f7a36b61/post-fs-data.sh#L18

**Comment by alicee98** [comment]
> hmmm, why do you remove magisk logs here?
> 
> https://github.com/alicee98/Selinux_Changer/blob/adfe2f29663cf2474fd80f6e22bc31e4f7a36b61/post-fs-data.sh#L18

Yeah I Think Its A Cache File. Are you wanna i delete it from the modules?

**Comment by HuskyDG** [comment]
it is benefit for debugging or send logs to magisk github

> > hmmm, why do you remove magisk logs here?
> > https://github.com/alicee98/Selinux_Changer/blob/adfe2f29663cf2474fd80f6e22bc31e4f7a36b61/post-fs-data.sh#L18
> 
> Yeah I Think Its A Cache File. Are you wanna i delete it from the modules?

It should not be removed because it is benefit for debugging and sending to Magisk github

**Comment by alicee98** [comment]
> it is benefit for debugging or send logs to magisk github
> 
> > > hmmm, why do you remove magisk logs here?
> > > https://github.com/alicee98/Selinux_Changer/blob/adfe2f29663cf2474fd80f6e22bc31e4f7a36b61/post-fs-data.sh#L18
> > 
> > 
> > Yeah I Think Its A Cache File. Are you wanna i delete it from the modules?
> 
> It should not be removed because it is benefit for debugging and sending to Magisk github

Okey I will remove it and fix it in different version

**Comment by alicee98** [comment]
> it is benefit for debugging or send logs to magisk github
> 
> > > hmmm, why do you remove magisk logs here?
> > > https://github.com/alicee98/Selinux_Changer/blob/adfe2f29663cf2474fd80f6e22bc31e4f7a36b61/post-fs-data.sh#L18
> > 
> > 
> > Yeah I Think Its A Cache File. Are you wanna i delete it from the modules?
> 
> It should not be removed because it is benefit for debugging and sending to Magisk github

Its done sir. Here's the link : https://github.com/alicee98/Selinux_Changer/releases/tag/V2.5.1

**Comment by Atrate** [comment]
Why does your module clear applications' caches? I get the potential benefits, but that's not the module's core functionality.

https://github.com/alicee98/Selinux_Changer/blob/ab0b8ef420f8991033a08f402f484a11e618d186/post-fs-data.sh#L14

**Comment by alicee98** [comment]
> Why does your module clear applications' caches? I get the potential benefits, but that's not the module's core functionality.
> 
> https://github.com/alicee98/Selinux_Changer/blob/ab0b8ef420f8991033a08f402f484a11e618d186/post-fs-data.sh#L14

Okey i will remove it

**Comment by yzyhk904** [comment]
> The good way is do not permissive `untrusted_app` `app_zygote` `isolated_app` domain

Yeah, Yeah. Do you mean that all processes including system processes should be inserted into untrusted app domain? The problem is that all SELinux enforced processes (running in parallel with music related apps) generate jitter and degrade digital audio quality, but not SELinux permissive processes.

**Comment by Atrate** [comment]
@alicee98 Why is `service.sh` included in the repo if it does nothing?
https://github.com/alicee98/Selinux_Changer/blob/main/service.sh

**Comment by Atrate** [comment]
I mean, it's not a problem, but if it's empty, it's useless
